nim/basolato/public/basolato.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
is this file needed ?

Reply:
Actuary it's not necessary to run app. Should I remove all files which are not called and make project minimal?

Reply:
Yes please

Reply:
@itsumura-h any time for this ?